cardano-node/src/Cardano/Node/Tracing/Tracers/Diffusion.hs

```diff
@@ -151,146 +136,127 @@ instance (LogFormatting peer, Show peer) =>
-              anyProto anyProto anyProto))
+        ["ExceptionExit"]
         []
         "Miniprotocol terminated with exception."
     , DocMsg
-        (WithMuxBearer peer
-          (MuxTraceChannelRecvStart anyProto))
+        ["ChannelRecvStart"]
         []
         "Channel receive start."
     , DocMsg
-        (WithMuxBearer peer
-          (MuxTraceChannelRecvEnd anyProto 1))
+        ["ChannelRecvEnd"]
         []
         "Channel receive end."
     , DocMsg
-        (WithMuxBearer peer
-          (MuxTraceChannelSendStart anyProto 1))
+        ["ChannelSendStart"]
         []
         "Channel send start."
     , DocMsg
-        (WithMuxBearer peer
-          (MuxTraceChannelSendEnd anyProto))
+        ["ChannelSendEnd"]
         []
         "Channel send end."
     , DocMsg
-        (WithMuxBearer peer
-          MuxTraceHandshakeStart)
+        ["HandshakeStart "]
         []
         "Handshake start."
     , DocMsg
-        (WithMuxBearer peer
-          (MuxTraceHandshakeClientEnd anyProto))
+        ["HandshakeClientEnd"]
         []
         "Handshake client end."
     , DocMsg
-        (WithMuxBearer peer
-          (MuxTraceHandshakeClientError protoSomeException anyProto))
+        ["HandshakeServerEnd"]
+        []
+        "Handshake server end."
+    , DocMsg
+        ["HandshakeClientError"]
         []
         "Handshake client error."
     , DocMsg
-        (WithMuxBearer peer
-          (MuxTraceHandshakeServerError protoSomeException))
+        ["HandshakeServerError"]
         []
         "Handshake server error."
     , DocMsg
-        (WithMuxBearer peer
-          MuxTraceSDUReadTimeoutException)
+        ["RecvDeltaQObservation"]
+        []
+        "Bearer DeltaQ observation."
+    , DocMsg
+        ["RecvDeltaQSample"]
+        []
+        "Bearer DeltaQ sample."
+    , DocMsg
+        ["SDUReadTimeoutException"]
         []
         "Timed out reading SDU."
     , DocMsg
-        (WithMuxBearer peer
-          MuxTraceSDUWriteTimeoutException)
+        ["SDUWriteTimeoutException"]
         []
         "Timed out writing SDU."
     , DocMsg
-        (WithMuxBearer peer
-          (MuxTraceStartEagerly anyProto anyProto))
+        ["StartEagerly"]
         []
         "Eagerly started."
     , DocMsg
-        (WithMuxBearer peer
-          (MuxTraceStartOnDemand anyProto anyProto))
+        ["StartOnDemand"]
         []
         "Preparing to start."
     , DocMsg
-        (WithMuxBearer peer
-          (MuxTraceStartedOnDemand anyProto anyProto))
+        ["StartedOnDemand"]
         []
         "Started on demand."
     , DocMsg
-        (WithMuxBearer peer
-          (MuxTraceTerminating anyProto anyProto))
+        ["Terminating"]
         []
         "Terminating."
     , DocMsg
-        (WithMuxBearer peer
-          MuxTraceShutdown)
+        ["Shutdown"]
         []
         "Mux shutdown."
-  ]
+    , DocMsg
+        ["Terminating"]
+        []
+        "Terminating."
```

Comment:
Is the period at the end of string intended?

Reply:
Yes, did it for documentation, in this case the documentation is somehow minimal